### DISCUSSION TITLE
Battery Health missing all AC charging data but shows up in LFP Battery Health?

### DISCUSSION THREAD

**Discussion by AlGahara2** [teslamate-org/teslamate#3913] [Q&A]
The Battery Health dashboard was fine before but started acting up since an update in April I think, where it would miss all AC charging data and started reporting unrealistically high battery capacity, like 102 kwh for my Model S 100D. This was ~93 kwh for months prior. Decided to look into it today and did another update, which fixed the abnormally high capacity numbers but it is still missing all the AC charging sessions in normal Battery Health Dashboard:
![image](https://github.com/teslamate-org/teslamate/assets/170750873/27e39986-4c84-49f7-af05-1e11f083f6c7)

In comparison, the LFP dashboard has all sessions including AC charging for the last month, so the data is there:
![image](https://github.com/teslamate-org/teslamate/assets/170750873/8cbef3e0-3636-44d2-9930-9dd463e64c4e)

All of these AC sessions are home charging lasting hours long and fits the criteria for Battery Health charging session. Any suggestions?

**Comment by jheredianet** [comment]
Be aware that this core dashboard, only considers charges for more than 10 minutes, more than Efficiency (let's say 15kWh) and less than 95%, if your charge sessions are shorts, they are not being considered for the estimations to avoid to avoid dirtying the samples.
About the LFP one, it's a dashboard that is outdated, so the core one is more accurate

**Reply by AlGahara2** [reply]
Ah that "more than Efficiency ~15kWh" is probably why, I don't drive too much and the home charging sessions are always below 15kWh per session, superchargers are used for longer trips that deplete over 15kWh of the battery which makes it look like none of the AC sessions are shown here, didn't realize that was part of the criteria for battery health dashboard too. How is this the charged capacity related to "Efficiency" though? Thank you for that note!

**Reply by jheredianet** [reply]
I you go to the efficiency dashboard, you will see the efficiency of your car (in the bottom left corner). That's the exact value that a charge is considered to get the estimations. 
In my case for example is 136 Wh/km, so if I would need to charge more than 13.6 kWh
![image](https://github.com/teslamate-org/teslamate/assets/45735191/e39c6a48-3697-49ec-b82b-34f0b429ed70)
